client/components/Auth/AuthAlert.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div
+    v-if="Object.keys(errors).length !== 0"
+    class="bg-red p-3 my-6 text-neutral-100 font-bold rounded-md"
+  >
+    <font-awesome-icon :icon="['fas', 'circle-exclamation']" />
+    <p class="inline">
+      <slot> Please fix the following errors: </slot>
+    </p>
+    <div v-for="(messages, error) in errors" :key="error" class="mt-2 text-lg">
+      {{ error }}
+      <p v-for="(message, i) in messages" :key="i" class="text-base">
+        {{ message }}
+      </p>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    errors: {
+      type: Object,
```

Comment:
Is this type correct? 

From the documentation you added in this PR it seems like this should be `type: Array`

Reply:
It's kinda both.
Each error is a property of an object.
This is the structure Axios uses for it's errors.

I'm working on a better type for this.
Because, I think it wasn't very modular to take an Axios error as a prop specifically.